### PR TITLE
Port all TestExpectations

### PR DESCRIPTION
**Pull request by KyleJu**
web-platform-tests/wpt.fyi#1993

### CODE REVIEW

File: css/compositing/mix-blend-mode/META.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+links:
+- product: chrome
+  url: crbug.com/1044742
```
Comment (stephenmcgruer) [inline css/compositing/mix-blend-mode/META.yml:3]: This isn't the best bug in the world (its a WPT import please-triage bug), but at least its open and the tests seem related (so sgtm).

File: css/css-animations/META.yml [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
   results:
-  - subtest: CSS Animations targetting (pseudo-)elements should have correct order
+  - test: Document-getAnimations.tentative.html
+    subtest: CSS Animations targetting (pseudo-)elements should have correct order
       after sorting (::marker)
-    test: Document-getAnimations.tentative.html
-  url: https://bugzilla.mozilla.org/show_bug.cgi?id=1635063
\ No newline at end of file
+- product: chrome
+  url: crbug.com/849859
```
Comment (stephenmcgruer) [inline css/css-animations/META.yml:13]: This is a WontFix

File: css/css-backgrounds/background-attachment-local/META.yml [hunk 3]
```diff
@@ -35,3 +35,8 @@ links:
     status: FAIL
   - test: attachment-local-positioning-4.html
     status: FAIL
+- product: chrome
+  url: crbug.com/771643
```
Comment (stephenmcgruer) [inline css/css-backgrounds/background-attachment-local/META.yml:39]: This is marked Fixed.

File: css/css-images/META.yml [hunk 4]
```diff
@@ -1,56 +1,61 @@
...
 - product: firefox
-  results:
-  - subtest: 'CSS namespace: attribute elementSources'
-    test: idlharness.html
   url: https://bugzilla.mozilla.org/show_bug.cgi?id=1634399
+  results:
+  - test: idlharness.html
+    subtest: 'CSS namespace: attribute elementSources'
+- product: chrome
+  url: crbug.com/604875
```
Comment (stephenmcgruer) [inline css/css-images/META.yml:58]: This is marked WontFix with a claim that the test works.

File: css/css-multicol/META.yml [hunk 5]
```diff
@@ -18,3 +18,50 @@ links:
...
+  url: crbug.com/874051
+  results:
+  - test: multicol-span-all-fieldset-001.html
+    status: FAIL
+  - test: multicol-span-all-fieldset-002.html
+    status: FAIL
+  - test: multicol-span-all-fieldset-003.html
+    status: FAIL
+- product: chrome
+  url: crbug.com/915204
```
Comment (stephenmcgruer) [inline css/css-multicol/META.yml:43]: Nit; this is duped into crbug.com/577282, might be worth changing to that.

File: css/css-multicol/META.yml [hunk 6]
```diff
@@ -18,3 +18,50 @@ links:
...
+  - test: multicol-span-all-007.html
+    status: FAIL
+  - test: multicol-span-all-009.html
+    status: FAIL
+  - test: multicol-span-all-008.html
+    status: FAIL
+  - test: multicol-span-all-011.html
+    status: FAIL
+- product: chrome
+  url: crbug.com/892817
```
Comment (stephenmcgruer) [inline css/css-multicol/META.yml:58]: Nit; this is duped into crbug.com/829028, might be worth changing to that.

File: css/css-shapes/shape-outside/values/META.yml [hunk 7]
```diff
@@ -1,12 +1,27 @@
...
+  - test: shape-margin-001.html
+    subtest: 10vh - computed
+  - test: shape-margin-001.html
+    subtest: 10vw - computed
+  - test: shape-margin-001.html
+    subtest: 10vmin - computed
+  - test: shape-margin-001.html
+    subtest: 10vmax - computed
+- product: chrome
+  url: crbug.com/441840
```
Comment (stephenmcgruer) [inline css/css-shapes/shape-outside/values/META.yml:14]: This is closed as Archived.

File: css/css-text-decor/META.yml [hunk 8]
```diff
@@ -1,69 +1,74 @@
...
+    status: FAIL
+  - test: text-decoration-underline-position-vertical.html
+    status: FAIL
 - product: firefox
+  url: https://bugzilla.mozilla.org/show_bug.cgi?id=1643421
   results:
   - test: text-decoration-thickness-from-font-variable.html
-  url: https://bugzilla.mozilla.org/show_bug.cgi?id=1643421
+- product: chrome
+  url: crbug.com/636993
```
Comment (stephenmcgruer) [inline css/css-text-decor/META.yml:71]: This is marked Fixed.

File: service-workers/service-worker/META.yml [hunk 9]
```diff
@@ -37,3 +37,18 @@ links:
...
+  results:
+  - test: ready.https.window.html
+    status: TIMEOUT
+- product: chrome
+  url: crbug.com/595993
+  results:
+  - test: fetch-header-visibility.https.html
+    status: FAIL
+- product: chrome
+  url: crbug.com/933880
```
Comment (stephenmcgruer) [inline service-workers/service-worker/META.yml:51]: This is marked Fixed.

File: web-animations/timing-model/timelines/META.yml [hunk 10]
```diff
@@ -0,0 +1,11 @@
+links:
+- product: chrome
+  url: crbug.com/626703
+  results:
+  - test: update-and-send-events.html
+    status: FAIL
+- product: chrome
+  url: crbug.com/912362
```
Comment (stephenmcgruer) [inline web-animations/timing-model/timelines/META.yml:8]: Duped into crbug.com/898176, may be worth replacing with that.

**Comment by KyleJu** [comment]
> canonical format

Yea I could probably arrange them in a sorting order. I will add it to Triage Metadata API too

**Comment by KyleJu** [comment]
@stephenmcgruer Applied all suggestions and created a [blacklist](https://github.com/web-platform-tests/wpt-metadata/pull/276/files#diff-18d16cec9192526d4a0131fa42182e0eR1), in case we will port TestExpectations continously in the future

**Pull request merged**